api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -61,6 +62,7 @@ pub trait Rpc {
 		channel_metadata: Option<CcmChannelMetadata>,
 		boost_fee: Option<BasisPoints>,
 		affiliate_fees: Option<Affiliates<AccountId32>>,
+		refund_parameters: Option<ChannelRefundParameters>,
```

Comment:
Good point, I forgot to do parameter validation. Will limit to 120 blocks (which would result in ~1h of retries given that we retry every 5 blocks).

Reply:
That seems pretty long imo. I think I'd rather err on a shorter length of time, and then increase it if it seems necessary. Just because shorter means less protocol work

Reply:
@dandanlen any thoughts on this one?

Reply:
Yeah was just chatting to Max in dms - agree 10 mins is too short. The variable I'm mostly concerned about here is protocol work. There's ofc the possibility to extend total duration without increasing protocol work. For example, by reducing the *frequency* of retries e.g. to 10 blocks, then we can double the TTL of the order.